bags-client.sln

```diff
@@ -1,22 +1,22 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "bags-client", "bags-client.xproj", "{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "bags-client", "bags-client.xproj", "{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{6BE8394A-B198-4B1B-80F5-8EB947F3C3F5}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+EndGlobal

```

Comment:
Actually, it appears this file may have had the wrong (or right depending on how you look at it) line endings previously... I'll investigate and figure out what the right answer is but in the meantime you should still remove this from the change set.